### PR TITLE
[SCP-2556] Fix focus on contract templates

### PR DESCRIPTION
**Pull request by jhbertra**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: marlowe-dashboard-client/src/InputField/View.purs [hunk 1]
```diff
@@ -7,16 +7,27 @@ import Data.Foldable (foldMap)
...
 
+inputCss :: InputDisplayOptions -> Boolean -> Array String
+inputCss { readOnly: true } = Css.inputNoFocus
+
+inputCss _ = Css.input
+
+getTabIndex :: InputDisplayOptions -> Int
+getTabIndex { readOnly: true } = -1
+
+getTabIndex _ = 0
```
Comment (hrajchert) [inline marlowe-dashboard-client/src/InputField/View.purs:29]: not too familiar with tab index, but shouldn't we specify with incremental order? 0, 1, 2, etc?

**Review by hrajchert — APPROVED** [review]
LGTM, I'll merge

File: marlowe-dashboard-client/src/InputField/View.purs (hunk 1 above)
Comment (jhbertra) [reply marlowe-dashboard-client/src/InputField/View.purs:29]: tab index `0` means the element will use the default tab order assigned by the browser. `-1` means that it is not focusable by tabbing.

Comment (jhbertra) [reply marlowe-dashboard-client/src/InputField/View.purs:29]: We could make this an ADT to make it more explicit...

Comment (jhbertra) [reply marlowe-dashboard-client/src/InputField/View.purs:29]: But that might make the semantics difficult to line up with... I think leaving as `Int` is the best, the semantics are web standard and generally well undeerstood https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

**Pull request merged**